src/app/Scenes/Onboarding/OnboardingSocialPick.tsx

```diff
@@ -89,35 +92,46 @@ export const OnboardingSocialPick: React.FC<OnboardingSocialPickProps> = ({ mode
       handleErrorWithAlternativeProviders(error.meta)
       return
     }
-    Alert.alert("Try again", error.message)
+    GlobalStore.actions.auth.setState({ sessionState: { isLoading: false } })
+    InteractionManager.runAfterInteractions(() => {
+      Alert.alert("Try again", error.message)
+    })
   }
 
+  const handleSocialLogin = async (callback: any) => {
```

Comment:
```suggestion
  const handleSocialLogin = async (callback: () => Promise<AuthPromiseResolveType>) => {
```
I think you might need to export `AuthPromiseResolveType` from `AuthModel.ts`

Reply:
good catch, i had that stashed and forgot to push it, i will push it when i fix the tests